### PR TITLE
autocheck tapped ticket for transfer

### PR DESCRIPTION
**Pull request by jwietelmann**
connect #455

### CODE REVIEW

File: src/Tickets/transferTickets.js [hunk 1]
```diff
@@ -194,12 +183,11 @@ export default class TransferTickets extends Component {
...
-    const {checkboxes,scannedEmail } = this.state
-    const {
-      state: {showQRModal},
-    } = this
-    let disabled = !this.validRecipient() || this.transferCount() < 1;
-    let disabledText = !this.validRecipient() ? "Valid Recipient Required" : this.transferCount() < 1 ? "No Tickets Selected" : "";
+    const {checkboxes, showQRModal} = this.state
+
+    let disabled = !this.validRecipient() || this.transferCount < 1
+    let disabledText = !this.validRecipient() ? "Valid Recipient Required" : this.transferCount < 1 ? "No Tickets Selected" : ""
```
Comment (daybreaker) [inline src/Tickets/transferTickets.js]: should validRecipient() be a `get` also? It's not being passed as a prop so I dont think it needs to be an arrow function to preserve context?

Also not a fan of a ternary nested in a ternary.

File: src/Tickets/transferTickets.js [hunk 2]
```diff
@@ -154,15 +150,8 @@ export default class TransferTickets extends Component {
     return this.state.emailOrPhone != ''
   }
 
-  transferCount = () => {
-    const {checkboxes} = this.state
-
-    return Object.keys(checkboxes).reduce((acc, id) => {
```
Comment (daybreaker) [inline src/Tickets/transferTickets.js]: I had to git blame this to make sure I didnt write it. *wipes brow in relief*

Comment (jwietelmann) [reply src/Tickets/transferTickets.js]: 💯 Trying to piece together the evidence of why my brain made me do this: I _think_ that before I made the change to `setChecked` I had to filter out the `false` checkbox values.

File: src/Tickets/transferTickets.js (hunk 1 above)
Comment (jwietelmann) [reply src/Tickets/transferTickets.js]: I wasn't a fan either. I guess I can go back and refactor it. I was just getting to that point where I had already made some pretty significant refactors and didn't want to press my luck.

**Pull request merged**